cobbler/modules/managers/bind.py

```diff
@@ -108,7 +96,7 @@ def __expand_IPv6(self, address):
                 fullAddress = fullAddress[:-1]
         groups = fullAddress.split(":")
         for group in groups:
-            while len(group) < validGroupSize:
+            while (len(group) < validGroupSize):
```

Comment:
Simple not needed afaik.

```suggestion
            while len(group) < validGroupSize:
```